### PR TITLE
SAS-76, SAS-66, SAS-51 # fixed dekel errors

### PR DESCRIPTION
**Pull request by MrBahur**

### CODE REVIEW

**Review by AmitNachimovitz — COMMENTED** [review]
I think u must add to your classes names "UT, IT, and AT" accordingly. Otherwise, I'm not sure our automation will recognize it.

**Review by AmitNachimovitz — CHANGES_REQUESTED** [review]
For the changes I mentioned earlier.

File: src/main/java/com/SAS/crudoperations/CRUD.java [hunk 1]
```diff
@@ -306,14 +324,31 @@ public static Object getTable(League league, Season season) {
      * @param referees
      * @return
      */
-    public static boolean addRefereesToLeague(League league, Season season, List<Referee> referees) {
+    public static boolean addRefereesToLeagueInSeason(League league, Season season, List<Referee> referees) {
```
Comment (Chen2908) [inline src/main/java/com/SAS/crudoperations/CRUD.java:327]: I would prefer my real name be left out..

File: src/test/java/com/SAS/LeagueManagement/assignRefereesToLeagueInSpecificSeasonBottomUp.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        return CRUD.isSeasonExist(season);
+    }
+    public static boolean addRefereesToLeagueInSeasonStub(League league, Season season, List<Referee> referees) {
+    return CRUD.addRefereesToLeagueInSeason(league,season,referees);
+    }
+
+    @Test
+    public void initLeagueTester() {
+        League league = new League("Ligat Ha'al");
+        Assertions.assertTrue(isLeagueExistStub(league));
```
Comment (Chen2908) [inline src/test/java/com/SAS/LeagueManagement/assignRefereesToLeagueInSpecificSeasonBottomUp.java]: Whats the difference between this and assertTrue(.....)?
We used assertTrue and imported import static org.junit.Assert.*;

File: src/test/java/com/SAS/LeagueManagement/initLeagueBottomUp.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        return CRUD.addLeague(league);
+    }
+    public static boolean isLeagueExistStub(League league) {
+        return CRUD.isLeagueExist(league);
+    }
+    @Test
+    public void initLeagueTester() {
+        League league= new League("Ligat Ha'al");
+        Assertions.assertTrue(addLeagueStub(league));
+        Assertions.assertTrue(isLeagueExistStub(league));
```
Comment (Chen2908) [inline src/test/java/com/SAS/LeagueManagement/initLeagueBottomUp.java]: Same here (and in other places)

**Review by yaarm — APPROVED** [review]

**Review by AmitNachimovitz — APPROVED** [review]

**Pull request merged**